lib/fluent/plugin/output.rb

```diff
@@ -435,6 +435,12 @@ def configure(conf)
         self
       end
 
+      def keep_buffer_config_compat
+        # Need this to call `@buffer_config.disable_chunk_backup` just as before,
+        # since some plugins may use this option in this shape.
```

Comment:
> in this shape

in this way?

Reply:
Thanks! Was it strange English? I will fix this.

Reply:
Fixed. Thanks!

Reply:
> Thanks! Was it strange English? I will fix this.

Yes. Normally, "(how to) do it like this" can be replaced with "in this way". o/

Reply:
I see! Thanks!